### PR TITLE
Update Simplified Chinese localization

### PR DESCRIPTION
**Pull request by Jerry23011**
### Changes
- Add new translations

@nixzhu here comes the new ones :)

### CODE REVIEW

File: IceCubesApp/Resources/Localization/zh-Hans.lproj/Localizable.strings [hunk 1]
```diff
@@ -122,21 +122,21 @@
...
-"settings.support.supporter.subtitle" = "By giving a monthly tip to Ice Cubes, you become a supporter, you'll even get verified badge on your profile. For your 👀 only.";
-"settings.support.supporter.subscribed" = " Thanks for being an Ice Cubes supporter!";
-"settings.support.supporter.subscription-info"= "Subscription are automatically renewed unless you cancel them a day before the next renewal cycle.";
-"settings.support.restore-purchase.button" = "Restore purchase";
-"settings.support.restore-purchase.explanation" = "Restore your supporter subscription if it's haven't been automatically synced.";
-"settings.support.privacy-policy" = "Privacy Policy";
-"settings.support.terms-of-use" = "Terms";
+"settings.support.supporter.title" = "成为支持者！";
+"settings.support.supporter.subtitle" = "你可以通过每月给 Ice Cubes 捐赠成为支持者，你还能在个人资料上得到一个认证徽章，仅你可见 👀";
+"settings.support.supporter.subscribed" = "感谢成为 Ice Cubes 的支持者！";
```
Comment (nixzhu) [inline IceCubesApp/Resources/Localization/zh-Hans.lproj/Localizable.strings:127]: “感谢”后加个“你”感觉合理一点

File: IceCubesApp/Resources/Localization/zh-Hans.lproj/Localizable.strings [hunk 2]
```diff
@@ -122,21 +122,21 @@
...
+"settings.support.supporter.subscription-info"= "订阅将会自动续订，除非在当前订阅期至少提前一天取消自动续订。";
```
Comment (nixzhu) [inline IceCubesApp/Resources/Localization/zh-Hans.lproj/Localizable.strings:128]: 此订阅将……

File: IceCubesApp/Resources/Localization/zh-Hans.lproj/Localizable.strings (hunk 1 above)
Comment (Jerry23011) [reply IceCubesApp/Resources/Localization/zh-Hans.lproj/Localizable.strings:127]: 确实更合理

File: IceCubesApp/Resources/Localization/zh-Hans.lproj/Localizable.strings (hunk 2 above)
Comment (Jerry23011) [reply IceCubesApp/Resources/Localization/zh-Hans.lproj/Localizable.strings]: ok

**Pull request merged**